t/07.1-extract-perl.t

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use LedgerSMB::Sysconfig;
```

Comment:
From what I can see, this is here solely for the temp dir. Please don't use Sysconfig for that, instead, use the core module File::Spec's `tmpdir` method.

Reply:
Done